functions/src/definitions/common/mediaUtils.ts

```diff
@@ -60,20 +63,24 @@ async function getHash(buffer: Buffer) {
 }
 
 async function getSignedUrl(storageUrl: string) {
-  try {
-    const storage = admin.storage()
-    const [temporaryUrl] = await storage
-      .bucket()
-      .file(storageUrl)
-      .getSignedUrl({
-        action: "read",
-        expires: Date.now() + 60 * 60 * 1000,
-      })
-    return temporaryUrl
-  } catch (error) {
-    functions.logger.error(error)
-    // You can also log the error to an external monitoring service or send an alert to the developers
-    return null
+  if (runtimeEnvironment.value() === "PROD") {
+    try {
+      const storage = admin.storage()
+      const [temporaryUrl] = await storage
+        .bucket()
+        .file(storageUrl)
+        .getSignedUrl({
+          action: "read",
+          expires: Date.now() + 60 * 60 * 1000,
+        })
+      return temporaryUrl
+    } catch (error) {
+      functions.logger.error(error)
+      // You can also log the error to an external monitoring service or send an alert to the developers
+      return null
+    }
+  } else {
```

Comment:
To reduce the depth of the prod code, moving this up to early return can be used `if (runtimeEnvironment.value() !== "PROD") {` can reduce the use one 1 set of {}

Reply:
Done